com.ibm.wala.cast.python.ml/data/tensorflow.xml

```diff
@@ -55,6 +55,9 @@
         <new def="reshape" class="Ltensorflow/functions/reshape" />
         <putfield class="LRoot" field="reshape" fieldType="LRoot" ref="x" value="reshape" />
 
+        <new def="ones" class="Ltensorflow/functions/ones" />
```

Comment:
No, because this is when we are importing `tensorflow`, when we import `tensorflow.python.ops.array` we go to this one https://github.com/ponder-lab/ML/pull/33/files#diff-679e457ba33ce5658ad9444fd41a8839e363bac4c0a268b27a86bbaad0c22056R96

Reply:
If I change that, this collides with the one I put above, then it will fail.